drivers/usb/device/usb_dc_sam.c

```diff
@@ -432,12 +428,12 @@ int usb_dc_ep_check_cap(const struct usb_dc_ep_cfg_data * const cfg)
 			return -1;
 		}
 	} else if (ep_idx & BIT(0)) {
-		if (EP_ADDR2DIR(cfg->ep_addr) != USB_EP_DIR_IN) {
+		if (USB_EP_GET_DIR(cfg->ep_addr) != USB_EP_DIR_IN) {
```

Comment:
s/a
Check if `USB_EP_DIR_OUT`

Reply:
No, I will not change this logic here. Also not relevant to this PR.